robots.js

```diff
@@ -0,0 +1,69 @@
+// Global variables to hold list of all fighters, and user-selected fighter 1 and 2
+let fighters = {};
+let fighter1 = {};
+let fighter2 = {};
+
```

Comment:
It would be better to avoid globals by wrapping your JS files in anamespace using iifes